src/Components/ListenOn/ListenOn.css

```diff
@@ -1,17 +1,16 @@
 .listenOnContainer {
-    display: flex;
-    flex-direction: column;
-    max-width: 1140px;
-    margin: auto;
+    /* max-width: 1140px;
```

Comment:
it looks like everything inside of `.listenOnContainer` is commented out, if there are no styles in here that you need you can remove the class entirely